schemas/context/media-timed-asset-view-details.schema.json

```diff
@@ -26,6 +26,16 @@
           "type": "string",
           "description": "Name of the content player."
         },
+        "xdm:videoSegment": {
+          "title": "Video segment",
+          "type": "string",
+          "description": "The interval that describes the part of the content that has been viewed in minutes."
```

Comment:
Could we at least tie this to a standard, such as Media Fragments (https://www.w3.org/TR/media-frags/#naming-time)?

Reply:
This doesn't follow the Media Fragments standard, it's something of the format 
[start minute]-[end minute], where both bounds of the interval are compulsory. Added a regex pattern to the field to specify this.